extensions/mssql/src/connectionconfig/connectionDialogWebviewController.ts

```diff
@@ -510,6 +520,27 @@ export class ConnectionDialogWebviewController extends FormWebviewController<
                     state.connectionProfile,
                 );
 
+                // trying to parse an unsupported authentication type will result in a undefined authenticationType
+                if (!state.connectionProfile.authenticationType) {
+                    // Show error for unsupported authentication type
+                    const errorMessage = l10n.t(
+                        "Unsupported authentication type in connection string. Only SQL Login, Integrated, and Azure MFA authentication types are supported.",
```

Comment:
In general, if an error message says that an input in incorrect, it's good to have the error repeat the invalid input back to the user (in this case, the "bad" authentication type).